### ISSUE TITLE
Option to automatically save to the gallery

### ISSUE THREAD

**Issue by alxgsv** [caydey/ffshare#15]
Hey! Thank you for ffshare! Not sure if it's a right place for feature request.

User journey 1:
1. Compression ends
2. I pick wrong app to share
3. I go back to change my mind
4. Compression result lost

User journey 2:
1. Compression ends
2. I can't find an option to just save compressed file back to the gallery

Proposal:
Implement an option to save all compression results to gallery or some folder before sharing

**Comment by gbdomubpkm** [comment]
Indeed, there is currently no automation for this. But you can already share the created file with the file manager to save it to the location of your choice.

**Comment by alxgsv** [comment]
@gbdomubpkm thank you! Could you recommend one, please? Google's Files are not listed there. And I am not very familiar with file managers.

**Comment by gbdomubpkm** [comment]
> @gbdomubpkm thank you! Could you recommend one, please? Google's Files are not listed there. And I am not very familiar with file managers.

Personally, i like and use https://f-droid.org/en/packages/com.simplemobiletools.filemanager.pro/

**Comment by Borph** [comment]
Ack, this would be helpful. For now, I use https://f-droid.org/de/packages/xyz.myachin.saveto/

**Comment by jersa** [comment]
Nice and needed app with the potential to be great! I'm not seeing any file manager options to share to, either. I'm using Material Files, me.zhanghai.android.files.

**Comment by Morse6690** [comment]
The default file manager on my device doesn't show up on the share sheet. Having the option to save to device would be a helpful feature without needing to install a specific file manager.

**Comment by Cwpute** [comment]
While waiting for a feature within ffshare, a user in another suggested using [SaveToDevice](https://www.f-droid.org/de/packages/name.lmj001.savetodevice/) a tool that allows the user to save a file using the Share To… menu

**Comment by SmokeMonsterPacks** [comment]
Came here to ask for this feature too. Great app, it's a life saver for Bluesky
